packages/tk.rb

```diff
@@ -27,21 +28,27 @@ class Tk < Package
   def self.build
     FileUtils.chdir('unix') do
       if ARCH == 'x86_64'
-        system './configure',
-               "--prefix=#{CREW_PREFIX}",
-               "--libdir=#{CREW_LIB_PREFIX}",
-               "--mandir=#{CREW_PREFIX}/share/man",
-               "--with-tcl=#{CREW_LIB_PREFIX}",
-               '--enable-threads',
-               '--enable-64bit'
+        system "env CFLAGS='-flto=auto -pipe -fuse-ld=gold' \
+          CXXFLAGS='-flto=auto -pipe -fuse-ld=gold' \
+          LDFLAGS='-flto=auto' \
+          ./configure \
+          --prefix=#{CREW_PREFIX} \
+          --libdir=#{CREW_LIB_PREFIX} \
+          --mandir=#{CREW_PREFIX}/share/man \
+          --with-tcl=#{CREW_LIB_PREFIX} \
+          --enable-threads \
+          --enable-64bit"
       else
-        system './configure',
-               "--prefix=#{CREW_PREFIX}",
-               "--libdir=#{CREW_LIB_PREFIX}",
-               "--mandir=#{CREW_PREFIX}/share/man",
-               "--with-tcl=#{CREW_LIB_PREFIX}",
-               '--enable-threads',
-               '--disable-64bit'
+        system "env CFLAGS='-flto=auto -pipe -fuse-ld=gold' \
+          CXXFLAGS='-flto=auto -pipe -fuse-ld=gold' \
+          LDFLAGS='-flto=auto' \
+          ./configure \
+          --prefix=#{CREW_PREFIX} \
+          --libdir=#{CREW_LIB_PREFIX} \
+          --mandir=#{CREW_PREFIX}/share/man \
```

Comment:
Can the 3 previous lines be replaced by `#{CREW_OPTIONS}`?

Reply:
Yup. That works fine.